### ISSUE TITLE
Questions about vat_loss tendency

### ISSUE THREAD

**Issue by poncey** [9310gaurav/virtual-adversarial-training#4]
Hi, thanks for your great work.
I modified your codes about VAT and make it applicable for my networks. I don't know if it is correct so I plotted the VAT loss and found it was smoothly increase. Is it right? cause I found in your code that it finally output the KL divergence, which is the negative of LDS so that it should increase (well, actually I am not so sure about it, so I would like to ask for u about this). I want to know if there is some problems  about my code or my thoughts? 
In addition, here is the plotted vat_loss. the cyan line describes the tendency of output vat_loss.
![loss_result](https://user-images.githubusercontent.com/35760029/50819087-0438cf00-1364-11e9-99c7-15c4eaeacf3a.png)

**Comment by ghost** [comment]
Maybe you need more training times to see the tendency such as 500 epochs.

**Comment by ghost** [comment]
Here's the result for CIFAR10 with VAT loss after training 500 epochs. I didn't modify the original code.
![vat_result](https://user-images.githubusercontent.com/63961454/93371648-43cbf400-f885-11ea-8d59-869bab663cab.png)

**Comment by poncey** [comment]
Appreciated! What you demonstrated is crystal clear. And I have already solved the problem previously by adjusting the learning rates and enlarge the training epochs.
